int16 joy_l_lr
int16 joy_l_ud
int16 joy_r_lr
int16 joy_r_ud
